### PR TITLE
Fix import of `CustomTransferMap`

### LINKED ISSUE #454: Import of `CustomTransferMap` from Elegant is broken
The Elegant import does not produce valid `CustomTransferMap` currently. Due to Cheetah's 7-dimensional state-space, the (7,7)-entry of the transfer map has to be 1 but its currently 0. This leads to loss of functionallity for all affine effects downstream of the `CustomTransferMap`.

### PR DESCRIPTION
**Pull request by Hespe**
## Description

The import of `CustomTransferMap` is currently broken since the affine component is not passed along.

## Motivation and Context

- [x] I have raised an issue to propose this change (required for new features and bug fixes)
  - Closes #454 

## Types of changes

- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] Documentation (update in the documentation)

## Checklist

- [x] I have updated the changelog accordingly (**required**).
- [ ] My change requires a change to the documentation.
- [x] I have updated the tests accordingly (_required for a bug fix or a new feature_).
- [ ] I have updated the documentation accordingly.
  
  
- [x] I have reformatted the code and checked that formatting passes (**required**).
- [x] I have have fixed all issues found by `flake8` (**required**).
- [x] I have ensured that all `pytest` tests pass (**required**).
- [ ] I have run `pytest` on a machine with a CUDA GPU and made sure all tests pass (**required**).
- [x] I have checked that the documentation builds (**required**).

Note: We are using a maximum length of 88 characters per line.

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -18,6 +18,7 @@
 - Fix issue where `Dipole` with `tracking_method="bmadx"` and `angle=0.0` would output `NaN` values as a result of a division by zero (see #434) (@jank324)
 - Fix issue in CI space-charge tests (incorrect beam duration in non-relativistic case) (see #446) (@RemiLehe)
 - Fix issue that passing tensors with `requires_grad=True` does not result in gradient tracked particles when using `ParticleBeam.from_parameters` initialization (see #445) (@cr-xu)
+- Fix import of `CustomTransferMap` from Elegant. The affine phase-space component was previously not carried through (#455) (@Hespe)
```
Comment (jank324) [inline CHANGELOG.md:21]:
```suggestion
- Fix import of `CustomTransferMap` from Elegant. The affine phase-space component was previously not carried through (see #455) (@Hespe)
```

**Review by jank324 — APPROVED** [review]
